src/main/kotlin/com/hibob/academy/employeeFeedback/dao/models.kt

```diff
@@ -47,12 +44,12 @@ enum class AnonymityType {
         fun convertAnonymityTypeToString(anonymity: AnonymityType): String =
             anonymity.toString()
 
-        fun convertStringToAnonymityType(anonymity: String): AnonymityType =
+        fun convertToAnonymityType(anonymity: String): AnonymityType =
```

Comment:
why do you need both functions if the first one just calls this one?

Reply:
because i want one of them to be nullable and the other not to be

Reply:
talked about it offline. fixed the naming